app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,15 @@
 class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(params[:comment].permit(:message, :user_id))
+    redirect_to post_path(@post)
+  end
+
+  def update
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.find(params[:id])
+    @comment.likes = @comment.likes + 1
```

Comment:
could you write as `@comment.likes += 1`  ?

Reply:
we could but errors showed up at the start since we had to change likes[ null to 0]. so after making changes to DB, we haven’t actually refactored our code